bpmn-statistic/src/com/axonivy/utils/bpmnstatistic/utils/ProcessesMonitorUtils.java

```diff
@@ -205,7 +205,7 @@ private static void updateArrowByWorkflowProgress(Map<String, Node> nodeMap, Wor
 
   private static void updateNodeByAnalysisType(Node node, AnalysisType analysisType) {
     if (AnalysisType.FREQUENCY == analysisType) {
-      node.setLabelValue(String.valueOf(node.getRelativeValue()));
+      node.setLabelValue(String.valueOf(node.getFrequency()));
```

Comment:
Change the type of the LabelValue to double
![image](https://github.com/user-attachments/assets/ce6cdbbb-0ed0-4780-9d65-2df45608632a)